### DISCUSSION TITLE
Exporting CSV from Kibana Fails

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#7416] [Unsupported Versions] [answered]
All:

I need to export a csv of all the logs resulting from a kibana query.

In the SO Kibana interface there is an option under the three dots menu to export the logs to a csv file.  This seems to be exactly what I need.  

However, when I try the csv always fails to generate.  Some google-ing leads me to believe this is because the resulting csv file is too big.  

Can someone please tell me:

1)  How can I adjust the max csv file size so that I can generate the file I need

and / or

2)  Am I an idiot and there is a better way to do it, and how?

**Comment by dougburks** [comment, marked as answer]
Kibana has a setting called xpack.reporting.csv.maxSizeBytes:
https://www.elastic.co/guide/en/kibana/current/reporting-settings-kb.html#reporting-csv-settings

You should be able to set that as shown in our Kibana configuration documentation:
https://docs.securityonion.net/en/2.3/kibana.html#configuration

**Reply by UMHB-InfoSec** [reply]
Thank you Doug. 

I did a lot of reading and went down several rabbit holes over the weekend.

I apologize for not being more specific with my question.
